hieradata/common.yaml

```diff
@@ -219,6 +219,7 @@ profile::jenkinscontroller::jcasc:
       jnlp-maven-11-windows: jenkinsciinfra/inbound-agent-maven:jdk11-nanoserver
       jnlp-ruby: jenkinsciinfra/inbound-agent-ruby@sha256:221f96baa1957742728eb6d2769a691aea5721fc11ed9b05da810c5debe92115
       jnlp-maven-8: jenkinsciinfra/inbound-agent-maven@sha256:4082d59973b7686af97dd4847c46b8affb57e1742820cc1fc238f847efc85f3a
+      jnlp-maven-8-aio: jenkinsciinfra/jenkins-agent-ubuntu-20.04@sha256:3a490163d60ff134481964b816ecac919b7b8699189d86e0db634b8d36678306
```

Comment:
```suggestion
     jnlp-maven-all-in-one: jenkinsciinfra/jenkins-agent-ubuntu-20.04@sha256:3a490163d60ff134481964b816ecac919b7b8699189d86e0db634b8d36678306
```

Reply:
for now, the template is forcing the same image name and the same agent name https://github.com/jenkins-infra/jenkins-infra/blob/b1657c5bef338d51be415f69f6d925533ac00e06/dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb#L199 so it need to contain the 8, 11 and 17

Reply:
I've updated my suggestion accordingly.